testsuite/module-publish/src/packages/framework_usecases/sources/coin_example.move

```diff
@@ -0,0 +1,20 @@
+module 0xABCD::coin_example {
+    use std::signer;
+
+    struct ExampleCoin {}
+
+    fun init_module(sender: &signer) {
+        aptos_framework::managed_coin::initialize<ExampleCoin>(
+            sender,
+            b"Example Coin",
+            b"Example",
+            8,
+            false,
+        );
+    }
+
+    public entry fun mint_p(user: &signer, admin: &signer, amount: u64) {
```

Comment:
sorry but what does `_p` stand for and how does an entry function take two signers?

Reply:
I saw someone else use _p before, so I continued using it. it is for "payer" before we had a proper fee payer.

but basically here it is for indirection - we sign a transaction with two signers - user is used for seq_num and gas, and admin is a second signature, for authorization purposes